src/PactNet.AspNetCore/ProviderState/ProviderStateOptions.cs

```diff
@@ -0,0 +1,13 @@
+namespace PactNet.AspNetCore.ProviderState
+{
+    /// <summary>
+    /// Defines the provider state middleware options
+    /// </summary>
+    public class ProviderStateOptions
+    {
+        /// <summary>
+        /// Route of provider state endpoint
+        /// </summary>
+        public string RouteProviderState { get; set; }
```

Comment:
I think this should have a default value and we should have an overload in `ProviderStateMiddlewareExtensions` which just uses the defaults. That makes the simple API possible with sensible defaults:

```csharp
services.AddPactProviderState();
```